R/utils.R

```diff
@@ -0,0 +1,117 @@
+library(shiny)
+library(shinyGovstyle)
+library(RCurl)
+library(stringr)
+
+#' Custom disconnect message
+#'
+#'
+#' @description
+#' Create the html overlay panel to appear when RSConnect disconnects
+#'
+#' @param refresh
+#' @param links A list of mirros or alternative links to the dashboard
+#' @param publication_name The parent publication name
+#' @param publication_link The link to the publication on Explore Education Statistics
+#'
+#' @return
+#' @export
+#'
+#' @examples
+customDisconnectMessage <- function(refresh = "Refresh page",
+                                    links = sites_list,
+                                    publication_name = ees_pub_name,
+                                    publication_link = ees_publication) {
+  # Check links are valid
+
+  is_valid_sites_list <- function(sites){
+    lapply(str_trim(sites), startsWith, "https://department-for-education.shinyapps.io/")
```

Comment:
As commented for the publication link validation below as well, is it worth also flagging an error if the link provided is _exactly_ this one - as in there should be the dashboard specific part of the url as well?